pages/_document.js

```diff
@@ -1,27 +1,54 @@
 import Document, { Html, Head, Main, NextScript } from 'next/document'
+import { generateNonce } from '../lib/Utils.js'
```

Comment:
Suggestion: you don't actually need the `.js` when importing
```suggestion
import { generateNonce } from '../lib/Utils'
```